### PR TITLE
Issue #6: simplify using separate databases for custom ontologies.

### PR DESCRIPTION
**Pull request by rbrush**
Simplify using separate databases for ontologies for issue #6.

### CODE REVIEW

File: python/bunsen/mapping/__init__.py [hunk 1]
```diff
@@ -10,41 +10,59 @@ class for details.
 import collections
 import datetime
 
-def get_default_concept_maps(spark_session):
+def get_concept_maps(spark_session, database='ontologies'):
+    """
+    Returns a :class:`ConceptMaps` instance for the given database.
```
Comment (bdrillard) [inline python/bunsen/mapping/__init__.py]: Should this docstring include `:py` as a prefix to the `:class:ConceptMaps` reference?

File: python/bunsen/mapping/__init__.py [hunk 2]
```diff
@@ -10,41 +10,59 @@ class for details.
...
+    """
     jconcept_maps = spark_session._jvm.com.cerner.bunsen.mappings \
       .ConceptMaps.getEmpty(spark_session._jsparkSession)
 
     return ConceptMaps(spark_session, jconcept_maps)
 
-def get_default_value_sets(spark_session):
+def get_value_sets(spark_session, database='ontologies'):
+    """
+    Returns a :class:`ValueSets` instance for the given database.
```
Comment (bdrillard) [inline python/bunsen/mapping/__init__.py]: Same question as above.

File: python/bunsen/mapping/__init__.py [hunk 3]
```diff
@@ -10,41 +10,59 @@ class for details.
...
     jvalue_sets = spark_session._jvm.com.cerner.bunsen.mappings \
-      .ValueSets.getDefault(spark_session._jsparkSession)
+      .ValueSets.getFromDatabase(spark_session._jsparkSession, database)
 
     return ValueSets(spark_session, jvalue_sets)
 
-def get_empty_value_sets(spark_session):
+def create_value_sets(spark_session):
+    """
+    Creates a new, empty :class:`ValueSets` instance.
```
Comment (bdrillard) [inline python/bunsen/mapping/__init__.py]: Same question as above.

File: python/bunsen/mapping/__init__.py [hunk 4]
```diff
@@ -10,41 +10,59 @@ class for details.
...
       .ValueSets.getEmpty(spark_session._jsparkSession)
 
     return ValueSets(spark_session, jvalue_sets)
 
-def get_default_hierarchies(spark_session):
-  jhierarchies = spark_session._jvm.com.cerner.bunsen.mappings \
-      .Hierarchies.getDefault(spark_session._jsparkSession)
+def get_hierarchies(spark_session, database='ontologies'):
+    """
+    Returns a :class:`Hierarchies` instance for the given database.
```
Comment (bdrillard) [inline python/bunsen/mapping/__init__.py]: Same question as above.

File: python/bunsen/mapping/__init__.py [hunk 5]
```diff
@@ -10,41 +10,59 @@ class for details.
...
 
-  return Hierarchies(spark_session, jhierarchies)
+    return Hierarchies(spark_session, jhierarchies)
 
-def get_empty_hierarchies(spark_session):
-  jhierarchies = spark_session._jvm.com.cerner.bunsen.mappings \
-      .Hierarchies.getEmpty(spark_session._jsparkSession)
+def create_hierarchies(spark_session):
+    """
+    Creates a new, empty :class:`Hierarchies` instance.
```
Comment (bdrillard) [inline python/bunsen/mapping/__init__.py]: Same question as above.

File: python/bunsen/mapping/__init__.py (hunk 1 above)
Comment (rbrush) [reply python/bunsen/mapping/__init__.py]: I had thought the same, but it seems that the prefix is implied when in a doc string. I did verify the links showed up appropriately in the generated documentation.

**Comment by rbrush** [comment]
Merged. Thanks for the review, Alek!

**Pull request merged**